lumicks/pylake/tests/test_imaging_confocal/conftest.py

```diff
@@ -22,6 +22,32 @@ def reference_counts():
                                0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 0, 2,
                                1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 2, 1, 0, 0, 2,
                                0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 2, 0, 1, 0, 2])
+reference_image = np.array(
+        [[[2., 2., 2.],
+          [0., 0., 0.],
+          [1., 1., 1.],
+          [0., 0., 0.]],
+
+         [[0., 0., 0.],
+          [0., 0., 0.],
+          [0., 0., 0.],
+          [1., 1., 1.]],
+
+         [[0., 0., 0.],
+          [0., 0., 0.],
+          [0., 0., 0.],
+          [1., 1., 1.]],
+
+         [[0., 0., 0.],
+          [0., 0., 0.],
+          [0., 0., 0.],
+          [1., 1., 1.]],
+
+         [[2., 2., 2.],
+          [0., 0., 0.],
+          [1., 1., 1.],
+          [0., 0., 0.]]]
+    )
 # fmt: on
```

Comment:
Is real content really needed? Wouldn't a simple `np.ones` or `np.eye` suffice for testing this bug? Since you don't need to check against any content it just seems like adding a lot of lines you don't actually use.

Reply:
You are right, this is not necessary. Replaced the `reference_image` with `np.ones` with the same shape.